Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "CallWithDoxDialer",
```

Comment:
For Comms to decide, but if you wanted to pick a different/better name for this project and also use it for the package, now would be a good time. The newer Ads SDK is `ios-ads-sdk` and the package is `DoximityAdsSDK` so something like `DoximityCallWithDialer` (if people really like the current name) or `DoximityDialerSDK` (which give more room for future additions) might follow that.

Reply:
I agree that we should update the name to be more inline with the naming convention of our current SDKs.

Reply:
@dmallory I updated the SDK to be renamed to `DoximityDialerSDK` in this commit: https://github.com/doximity/CallWithDoxDialer/pull/22/commits/a93363a3978cd16c28abdf8c696b5c496c588c38